torch/_dynamo/variables/torch.py

```diff
@@ -409,6 +409,8 @@ def call_function(
                 ),
                 example_value=None,
             )
+        elif self.value.__name__ == 'sparse_bsc_tensor':
+            unimplemented("sparse_bsc_tensor")
```

Comment:
The correct way to to this is:
1, Add ```"torch.sparse_bsc_tensor": SkipFunctionVariable,``` to ```manual_torch_name_rule_map``` in ```trace_rules.py```;
2, Remove ```"torch.sparse_bsc_tensor"``` from ```torch_c_binding_in_graph_functions``` in ```trace_rules.py```.
And there are a few similar functions, probably you want have the same treatment:
https://github.com/pytorch/pytorch/blob/2240018c03744ee34ea14ad53481db934c37e384/torch/_dynamo/trace_rules.py#L1989-L1994

Reply:
thanks for the tip